### PR TITLE
add expectation to file

### PR DESCRIPTION
**Pull request by eveleighoj**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Add expectations

## Related Tickets & Documents

- Ticket Link
- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] Are there any dependencies on other PRs or Work?

### CODE REVIEW

File: bin/download_expectations.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+spec = Specification('specification')
+cache_exp_dir = 'var/expectations'
+
+datasets = spec.dataset
+for dataset_obj in datasets.values():
+    if dataset_obj['collection']:
+        s3="https://files.planning.data.gov.uk"
+        dataset = dataset_obj['dataset']
+        print(dataset)
```
Comment (ssadhu-sl) [inline bin/download_expectations.py:27]: left print statement by mistake?

**Review by ssadhu-sl — APPROVED** [review]
looks good. can see one print statement.

File: bin/download_expectations.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        # https://files.planning.data.gov.uk/log/expectations/dataset=ancient-woodland/ancient-woodland.parquet
```
Comment (CarlosCoelhoSL) [inline bin/download_expectations.py:28]: this should probably be removed

**Review by CarlosCoelhoSL — COMMENTED** [review]
lgtm just removing a comment and print

**Pull request merged**